peps/pep-9999.rst

```diff
@@ -1075,19 +1104,20 @@ Reference Implementation
 
 A reference implementation should include three components:
 
-1. A central registry that captures at a minimum a ``dep:`` identifier and its description. This registry MUST
+1. A central registry that captures at a minimum a DepURL and its description. This registry MUST
    NOT contain specifics of package ecosystem mappings.
 2. A standard specification for a collection of mappings. JSON Schema is widely used for schema
    in many text editors, and would be a natural choice for expression of the standard specification.
 3. An implementation of (2), providing mappings from the contents of the central
    registry to the ecosystem-specific package names.
 
-For (1), the JSON Schema is defined at https://github.com/jaimergp/external-metadata-mappings/blob/main/schemas/central-registry.schema.json.
-An example registry can be found at https://github.com/jaimergp/external-metadata-mappings/blob/main/data/registry.json.
-For (2), the JSON Schema is defined at https://github.com/jaimergp/external-metadata-mappings/blob/main/schemas/external-mapping.schema.json.
-For (3), a collection of example mappings for a sample of packages can be found at https://github.com/jaimergp/external-metadata-mappings/tree/main/data.
-
-The JSON Schemas are created with `this Pydantic model <https://github.com/jaimergp/external-metadata-mappings/blob/main/schemas/schema.py>`__.
+For (1), the JSON Schema is defined at `central-registry.schema.json <https://github.com/jaimergp/external-metadata-mappings/blob/main/schemas/central-registry.schema.json>`__.
+An example registry can be found at `registry.json <https://github.com/jaimergp/external-metadata-mappings/blob/main/data/registry.json>`__.
+For (2), the JSON Schema is defined at `external-mapping.schema.json <https://github.com/jaimergp/external-metadata-mappings/blob/main/schemas/external-mapping.schema.json>`__.
+A collection of example mappings for a sample of packages can be found at `external-metadata-mappings <https://github.com/jaimergp/external-metadata-mappings/tree/main/data>`__.
+For (3), the JSON Schema is defined at `known-ecosystems.schema.json <https://github.com/jaimergp/external-metadata-mappings/blob/main/schemas/known-ecosystems.schema.json>`__.
+An example list can be found at `known-ecosystems.json <https://github.com/jaimergp/external-metadata-mappings/blob/main/data/known-ecosystems.json>`__.
```

Comment:
To be honest, I wouldn't refer to this as (1), (2), (3) but instead name them explicitly. I can imagine these references might be an accessibility nightmare.